Engine/DataFeeds/LiveTradingDataFeed.cs

```diff
@@ -409,8 +409,12 @@ protected Subscription CreateSubscription(Universe universe, Security security,
                     var enqueable = new EnqueueableEnumerator<BaseData>();
                     _customExchange.SetDataHandler(config.Symbol, data =>
                     {
-                        enqueable.Enqueue(data);
-                        if (subscription != null) subscription.RealtimePrice = data.Value;
+                        // Only enqueue new data if there is none (LastEnqueued == null) or if newer 
+                        if (enqueable.LastEnqueued == null || data.EndTime > enqueable.LastEnqueued.EndTime)
+                        {
```

Comment:
This may not be the best place for filtering logic, this handler is intended to pipe data out of a data exchange. I also worry that needing to add this here just hides another bug somewhere else.

I think  we could get similar behavior if we modified to this [line](https://github.com/AlexCatarino/Lean/blob/quandl-issue373/Engine/DataFeeds/Enumerators/FastForwardEnumerator.cs#L71) in `FastForwardEnumerator` to filter out the equals case as well (or maybe allow it as a bool option, tick could allow the equals case, all others should not).